### PR TITLE
Fix Sunnarium

### PR DESCRIPTION
**Pull request by Ethryan**
[#7373](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7373)

### CODE REVIEW

**Comment by Ethryan** [comment]
Damn, wait, i will fix this, but i need to do it in my test environment

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -1674,6 +1674,7 @@ public void run2() {
         GT_Values.RA.addPulveriserRecipe(new ItemStack(Items.record_wait, 1, 0),  new ItemStack[]{GT_ModHandler.getModItem("Avaritia", "Resource", 9L, 7)}, new int[]{10000}, 300, 2);
 
         GT_Values.RA.addPulveriserRecipe(GT_ModHandler.getModItem("AdvancedSolarPanel", "asp_crafting_items", 1L, 0), new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sunnarium, 1L)}, new int[]{10000}, 300, 2);
+        GT_Values.RA.addPulveriserRecipe(GT_ModHandler.getModItem("AdvancedSolarPanel", "asp_crafting_items", 1L, 9), new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Sunnarium, 1L)}, new int[]{10000}, 30, 2);
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1677]: why tiny pile?

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 2]
```diff
@@ -1083,7 +1083,7 @@ public void run() {
 
         GT_Values.RA.addCompressorRecipe(GT_ModHandler.getModItem("IC2", "itemWeed", 16L), ItemList.IC2_Plantball.get(1L), 300, 2);
         GT_Values.RA.addCompressorRecipe(GT_ModHandler.getModItem("ExtraTrees", "food", 64L, 24), ItemList.IC2_Plantball.get(1L), 300, 2);
-        GT_Values.RA.addCompressorRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sunnarium, 4L), GT_ModHandler.getModItem("AdvancedSolarPanel", "asp_crafting_items", 1L, 0), 300, 2);
+        GT_Values.RA.addCompressorRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sunnarium, 1L), GT_ModHandler.getModItem("AdvancedSolarPanel", "asp_crafting_items", 1L, 0), 300, 2);
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1086]: why we not let it be 4 dust one sunnarium item/ball and change the other reverse recipes to be correct ?

**Review by Dream-Master — CHANGES_REQUESTED** [review]

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 1 above)
Comment (Ethryan) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1677]: 9 compressed sunnarium pieces for 1 sunnarium.

Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1677]: a miss readed it so nevermind

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 2 above)
Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1086]: ok so we change the ratio. The smaller one is a tiny pile and the larger one is on dust

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**